rafthttp/peer.go

```diff
@@ -161,8 +162,7 @@ func startPeer(tr http.RoundTripper, urls types.URLs, local, to, cid types.ID, r
 					if isMsgSnap(m) {
 						p.r.ReportSnapshot(m.To, raft.SnapshotFailure)
 					}
-					// TODO: log start and end of message dropping
-					plog.Warningf("dropping %s to %s since %s's sending buffer is full", m.Type, p.id, name)
+					status.dropMessage(m, fmt.Sprintf("sending buffer on %s is full", name))
```

Comment:
@xiang90 
My thought is that in normal mode, etcd tells users that he is dropping some messages now, and it cannot tolerate the errors so far, so users will have expectation that the member will do something to recover itself when reconnection.
But your method is a simpler one, and inactive log line has been a stronger signal to user. will change.